lib/crawly/engine.ex

```diff
@@ -242,12 +242,17 @@ defmodule Crawly.Engine do
 
   defp set_spider_log(spider_name, crawl_id) do
     log_dir = Crawly.Utils.get_settings(:log_dir, spider_name, "/tmp")
-    Logger.add_backend({LoggerFileBackend, :debug})
+    enable_file_logging = Crawly.Utils.get_settings(:enable_file_logging)
+    current_unix_timestamp = :os.system_time(:second)
 
-    Logger.configure_backend({LoggerFileBackend, :debug},
-      path: "/#{log_dir}/#{spider_name}/#{crawl_id}.log",
-      level: :debug,
-      metadata_filter: [crawl_id: crawl_id]
-    )
+    if enable_file_logging do
```

Comment:
to make it more concise, could you do:

```elixir
if Crawly.Utils.get_settings(:log_to_file) do
    log_dir = Crawly.Utils.get_settings(:log_dir, spider_name, "/tmp")
    current_unix_timestamp = :os.system_time(:second)
end
...

might be cleaner to move the log_to_file check to the main function call, removing the if-do

Reply:
Additionally, should use get_settings/2 instead,  should include the spider_name, as we'll want to check if there are spider-level overrides.

Reply:
Done